### PR TITLE
[ALLUXIO-3052] Allow multiple fuse mount points per machine

### PR DESCRIPTION
**Pull request by yupeng9**
This PR changes the fuse command to:
 - Mount: ` mount mount_point [alluxio_path]`. Mount an Alluxio path to a given mount point. When `alluxio_path` is not given, default to `/`
 - Unmount: `unmount mount_point`. Unmount the fuse from the given mount point
 - Status: `stat`. Displays the `pid,mount_point,alluxio_path` of each fuse

### CODE REVIEW

File: integration/fuse/bin/alluxio-fuse.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 SCRIPT_DIR="$(cd "$(dirname "$(readlink "$0" || echo "$0")")"; pwd)"
 
-get_env () {
+function getEnv {
```
Comment (apc999) [inline integration/fuse/bin/alluxio-fuse.sh]: how about following google bash style guide in naming?
https://google.github.io/styleguide/shell.xml#Function_Names

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 2]
```diff
@@ -129,7 +129,8 @@ public int chmod(String path, @mode_t long mode) {
    * Changes the owner of an Alluxio file.
    *
    * This operation only works when the group mapping service is shelled based and the user is
-   * registered in unix. Otherwise it errors as not implemented.
+   * registered in unix. Otherwise it errors as not implemented. This is because the input uid and
+   * gid must map to the user and group in Unix.
```
Comment (apc999) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: must match?

File: integration/fuse/bin/alluxio-fuse.sh (hunk 1 above)
Comment (yupeng9) [reply integration/fuse/bin/alluxio-fuse.sh]: I'm fine with the proposed style, but I changed it to be consistent with the style in `bin/alluxio`. 
In this case, I'll change the styles of all our scripts.

**Comment by yupeng9** [comment]
@aaudiber PTAL?

**Pull request merged**